velox/type/Type.cpp

```diff
@@ -713,3 +713,20 @@ exec::CastOperatorPtr getCastOperator(const std::string& name) {
 }
 
 } // namespace facebook::velox
+
+namespace folly {
+int64_t hashLongDecimal(const ::facebook::velox::int128_t value) {
```

Comment:
Are these missing from the folly library?

Reply:
I am getting linking issues so I did this work around.

Reply:
why not use `commutativeHashMix`. It is odd to define folly namespace methods here.

Reply:
Will use commutativeHashMix.